SwiftyJSON.xcodeproj/project.pbxproj

```diff
@@ -672,7 +684,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 7.0;
```

Comment:
The podspec minimum target for iOS 8.0 so I think it makes sense to change the Xcode settings, too. I didn't dig into whether the pod spec could be bumped down to 7.0.